specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/serverVulnerabilityAssessments.json

```diff
@@ -0,0 +1,351 @@
+        "tags": [
+          "ServerVulnerabilityAssessment"
+        ],
+        "operationId": "ServerVulnerabilityAssessment_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "name": "resourceNamespace",
+            "in": "path",
+            "description": "The Namespace of the resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceType",
+            "in": "path",
+            "description": "The type of the resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "description": "Name of the resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "serverVulnerabilityAssessment",
+            "in": "path",
+            "description": "ServerVulnerabilityAssessment status. only a 'default' value is supported.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ]
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content - Resource does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerVulnerabilityAssessmentsList": {
+      "type": "object",
+      "description": "List of server vulnerability assessments",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ServerVulnerabilityAssessment"
+          }
+        }
+      }
+    },
+    "ServerVulnerabilityAssessment": {
+      "type": "object",
+      "description": "Describes the server vulnerability assessment details on a resource",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ServerVulnerabilityAssessmentProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ServerVulnerabilityAssessmentProperties": {
+      "type": "object",
+      "description": "describes ServerVulnerabilityAssessment properties.",
+      "properties": {
+        "provisioningState": {
```

Comment:
what value does this resource type add when it just gives back the provisioning state property to the user?

Reply:
This provisions a vulnerability assessment scanner on a VM. once it is installed, the user can have scans and see results in Azure Security Center Assessments API.

Just noting that this API was already added ~year ago for preview- This is just a copy in a Stable API.